docs/UserGuide.md

```diff
@@ -162,6 +251,18 @@ Format: `edit -m <module_code> <new_module_code>`
 Example of usage: 
 * `edit -m CS2113 CS2113T`
 
+Example of output:
+```
+Command: 
+⋗	edit -m CG1112 CG2271
```

Comment:
Again for continuity (in the case that the user follow the examples in UG), CG1111 instead?